playwright/e2e/suites/accession.disable.ts

```diff
@@ -0,0 +1,158 @@
+import { test, expect } from '@playwright/test';
+
+const waitFor = async (page, selector, timeout = 3000) => {
+  // Seems weird to await in here with nothing else going on, but I am doing that explicitly so I can
+  // ditch the return and keep the signature Promise<void>
+  await page.waitForSelector(selector, { timeout });
+};
+
+//ADE not sure what this is for?
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
+
+test.setTimeout(60000);
+test.beforeEach(async ({ context }, testInfo) => {
+  // Make all requests look like they are associated with an existing login session
+  // so we don't have to depend on a Keycloak server to run the test suite. The
+  // session value here is the base64-encoded session ID from dump/session.sql.
+  await context.addCookies([
+    { name: 'SESSION', value: 'Mjc2NzE0YWQtYWIwYS00OGFhLThlZjgtZGI2NWVjMmU5NTBh', url: 'http://127.0.0.1:3000' },
```

Comment:
get the cookie from process.env?

Reply:
How do you get the cookie into the env?

Reply:
It may be feasible (not too sure may need to test).
Locally using `.env` file, in GitHub using secrets?
web-components does something similar for publishing npm packages https://github.com/terraware/web-components/blob/main/.github/workflows/publish-rc.yaml#L21
We would have to store the cookie in the github repo as a secret here https://github.com/terraware/terraware-web/settings/secrets/actions

Reply:
Ignore this, was trying to avoid storing cookies in the code, but we already have this in the db setup script. Please ignore the suggestion. Maybe move the addCookies setup to the utils file so there's one place to set the cookie if anything changes.

Reply:
Yeah this is also the same way we were doing the cookies for Cypress.